test/verify/check-storage-hidden

```diff
@@ -123,6 +126,32 @@ class TestStorageHidden(StorageCase):
         b.wait_visible("#storage")
         self.assertEqual(m.execute(f"grep {mount_point_2} /etc/fstab || true"), "")
 
+    @unittest.skipUnless("ubuntu" in testvm.DEFAULT_IMAGE, "Only test snaps on Ubuntu")
+    def testHiddenSnap(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        # We expect there to be at least one loop back device mounted
+        # somewhere below /snap.
+        snap_loops = []
+        devices = json.loads(m.execute("lsblk --json --list --output NAME,TYPE,MOUNTPOINT"))
+        for d in devices["blockdevices"]:
+            if d["type"] == "loop" and d["mountpoint"].startswith("/snap/"):
+                snap_loops.append(d["name"])
+        self.assertTrue(len(snap_loops) > 0)
+
+        # Make one more loopback device that we expect to see in the UI.
+        dev = self.add_loopback_disk()
+
+        # Now we wait until the regular loopback device is shown.  The
+        # snaps should not be shown.
+        b.wait_in_text("#others", dev)
+        for sl in snap_loops:
+            b.wait_not_in_text("#others", sl)
+            b.wait_not_in_text("#mounts", sl)
```

Comment:
This is a good check as well. I originally meant that the string `/snap` should not appear anywhere on the page. But good enough, if you are bored enough about this by now :grin:

Reply:
Cockpit wont unfortunately show the "/snap" mount point, since it is not in fstab but in a systemd mount unit...